katalon-start-page/release-highlights.md

```diff
@@ -1,8 +1,7 @@
-### [Highlights from 10.1.1 and 10.1.0](https://docs.katalon.com/docs/release-notes/katalon-studio/katalon-studio-release-notes-version-10.x)
+### [Highlights from 10.2.0](https://docs.katalon.com/docs/release-notes/katalon-studio/katalon-studio-release-notes-version-10.x)
 
-* [10.1.1]:
-  * [Azure DevOps Integration]: Officially supports parameterizing associated Azure DevOps (ADO) Test Case IDs. For more details, see [Parameterize Azure DevOps Test Case ID List in Katalon Studio](https://docs.katalon.com/katalon-platform/proof-of-concept/parameterize-azure-devops-test-case-id-list-in-katalon-studio-poc#ariaid-title1).
-  * [Azure DevOps Integration]: Added a new Execution Profile column in the Azure DevOps Test Configuration Mapping to improve test result accuracy and management.
-  * [Jira Integration] Introduced the ability to add a Jira issue directly to an existing test case, improving issue traceability and test case management.
-* [10.1.0]:
-  * Introduced [StudioAssist chat window](https://docs.katalon.com/katalon-studio/create-test-cases/studioassist-chat-window): you can now ask StudioAssist anything related to Studio and Katalon products.
+* Introduced StudioAssist Chat Window with Project-Based Context.
```

Comment:
* Introduced StudioAssist Chat Window with project-based context.
* Introduced new HTML test reports.
* Introduced Protected Global Variables for secure storage.
* Upgraded Eclipse to version 2024.06.
* Added the ability to use Kerberos Authentication for proxy environments.